qmlexport/CMakeLists.txt

```diff
@@ -22,7 +32,7 @@ if(BUILD_QSKDLL)
 endif()
 
 # packaging
-set(PACKAGE_NAME            QmlExport)
+set(PACKAGE_NAME            QSkinnyQmlExport)
```

Comment:
Consistent nomenclature e.g. `UpperCase` for exported QSkinny CMake targets e.g.

`QSkinnyQmlExport` or `QSkinnyUpcomingSomething`